tests/test_commcare_hq_client.py

```diff
@@ -154,7 +160,7 @@ def test_repeat_limit(self):
             self._test_iterate(FakeRepeatedDateCaseSession(), get_paginator('case', 2), 2, [1, 2])
 
     def test_message_log(self):
-        self._test_iterate(FakMessageLogSession(), get_paginator('messaging-event', 2), 2, [1, 2])
+        self._test_iterate(FakMessageLogSession(), get_paginator('messaging-event', 2), 3, [1, 2, 3])
```

Comment:
is this a spelling mistake? I know this is what the class is called, but it reads a bit funny.

Reply:
:eyes: 189865f